unions-intersections/Exercise2/problem/index.ts

```diff
@@ -0,0 +1,31 @@
+interface Mortgage {
+  interest: number;
+  principle: number;
+}
+
+interface NonMortgageExpenses {
+  propertyTax: number;
+  insurance: number;
+  hoa: number;
+}
+
+interface MonthlyExpenses {
+  electricity: number;
+  water: number;
+  gas: number;
+  savings: number;
+}
+
+type HomeMonthlyExpense = never; // Fix this!
```

Comment:
I think this is solved by both unions and intersections. Should we change this so that there’s only one correct answer?